src/gcodeExport.cpp

```diff
@@ -197,6 +201,7 @@ std::string GCodeExport::getFileHeader(const double* print_time, const std::vect
         else if (flavor == EGCodeFlavor::REPRAP)
         {
             prefix << ";Filament used: " << ((filament_used.size() >= 1)? filament_used[0] / (1000 * extruder_attr[0].filament_area) : 0) << "m" << new_line;
+            prefix << ";Layer height: " << layer_height << new_line;
```

Comment:
I don't understand your comment about file scope.

There is not neccessarily one `layer_height` used throughout the whole gcode, so I don't like to have that variable as a global variable.